server/services/communityApiService.ts

```diff
@@ -14,16 +14,38 @@ export interface DeliusUser {
 }
 
 export interface DeliusServiceUser {
-  offenderId: string
+  otherIds: OtherIds
+  offenderProfile: OffenderProfile
+  title: string
   firstName: string
   surname: string
   dateOfBirth: string
+  gender: string
+  ethnicity: string
+  religionOrBelief: string
+  disabilities: string
 }
 
 interface DeliusRole {
   name: string
 }
 
+interface OtherIds {
+  crn: string
+  nomsNumber: string
```

Comment:
heads up we have decided to de-scope these other ids for MVP. we're only storing CRN.

Reply:
Yeah, I'm following the structure for the object in Delius with this object, I left off some other fields, so could get rid of these if we're not going to use them (same with needs interpreter below)

Reply:
Have removed these fields we're not using at the moment.